e2e/nomostest/config-sync.go

```diff
@@ -363,12 +364,10 @@ func installationManifests(nt *NT, tmpManifestsDir string, nomos ntopts.Nomos) [
 			replaced = strings.ReplaceAll(replaced, "HELM_SYNC_IMAGE_NAME", helmSyncImgName)
 		case "admission-webhook.yaml":
 			imgName = fmt.Sprintf("%s/admission-webhook:%s", *e2e.ImagePrefix, *e2e.ImageTag)
+			replaced = strings.ReplaceAll(replaced, "WEBHOOK_IMAGE_NAME", imgName)
 		default:
 			// For any other template, we want the latest image for the nomos binary (mono-repo).
 			imgName = fmt.Sprintf("%s/nomos:%s", *e2e.ImagePrefix, *e2e.ImageTag)
-		}
-
-		if template != "reconciler-manager-configmap.yaml" {
```

Comment:
I think it would be better if we reused the make targets for constructing the manifests here
- to minimize duplicate logic
- to test our real `build-manifests` as part of our e2e testing

That's a bigger change though that can be done separately